cmds/core/pwd/pwd_test.go

```diff
@@ -0,0 +1,33 @@
+// Copyright 2012-2023 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+	"os"
+	"path/filepath"
+	"testing"
+)
+
+func TestPWD(t *testing.T) {
+	dir := t.TempDir()
+	err := os.Chdir(dir)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	rs, err := filepath.EvalSymlinks(dir)
+	if err != nil {
+		t.Fatal(err)
```

Comment:
In the error messages, please follow the more common style:
t.Fatal("EvalSymLinks(%s): got %v, want nil", dir, err)

it's easier to debug logs that way
Same for all the other error. Thanks for your may contributions to u-root!

Reply:
updated, thanks for kind words :)